ipfx/nwb_reader.py

```diff
@@ -318,24 +318,26 @@ def get_sweep_number(self, sweep_name):
 
     def get_stim_code(self, sweep_name):
 
-        stimulus_description = "aibs_stimulus_description"
+        names = ["aibs_stimulus_name", "aibs_stimulus_description"]
```

Comment:
I think Pipeline nwbs only use "aibs_stimulus_name"

Reply:
@sgratiy The file 595570553.nwb which I'm using from the celltypes database is from the pipeline and has `aibs_stimulus_name` see

```
h5dump 595570553.nwb | grep -A 3 stimulus_name
           DATASET "aibs_stimulus_name" {
              DATATYPE  H5T_STRING {
                 STRSIZE H5T_VARIABLE;
                 STRPAD H5T_STR_NULLTERM;
                 CSET H5T_CSET_UTF8;
                 CTYPE H5T_C_S1;
```

and https://github.com/AllenInstitute/ipfx/blob/fef94b981ed174fd7820b72abb353e3181937d72/tests/test_nwb_reader.py#L130.